docs/user/SUMMARY.md

```diff
@@ -2,6 +2,7 @@
 
 - [Using the Glean SDK](user/index.md)
     - [Adding Glean to your project](user/adding-glean-to-your-project.md)
+    - [Integrating Glean for project managers](user/integrating-glean-for-product-managers.md)
```

Comment:
Hmmmm, not perfectly happy with this being the second chapter (though maybe the doc re-org may solve that).
What do you think about keeping it in [Focused Use Cases](https://mozilla.github.io/glean/book/user/focused-use-cases.html) for now?

Reply:
Fine with me -- @brizental would appreciate your thoughts about this in the context of the planned reorg.

Reply:
This page is a guide, in prose format. As such, it fits on the "User's Guides" section of the re-structured book.

I agree with jan-erik this doesn't need to be so up front, I think we can put it at the end of the list of "User's Guides". If necessary that can evolve to a whole section on product specific guides. 

If possible to block this PR on the #1601  then we already put the page where it will live.

Reply:
Cool.  That makes sense.  This isn't so urgent that it can't wait for #1601 to go first.